client/src/pages/ChatPage.js

```diff
@@ -157,11 +163,15 @@ const ChatPage = () => {
       </div>
       <div className="p-5 !pl-[300px]">
         <div className="flex justify-center">
-          <Chip
-            value={chat.name ? chat.name : ""}
-            variant="outlined"
-            className="text-center text-indigo-300 w-3/5 text-ellipsis overflow-hidden"
-          />
+          <div className="w-4/5 text-ellipsis overflow-hidden">
+            <Typography className="font-sans text-black/80 font-semibold text-4xl">
+              {currChat && currChat.name}
+            </Typography>
+            <Typography className="text-blue-gray-400 font-sans text-md ml-1">
+              {currChat.users && currChat.users.length} members
```

Comment:
As a very small, far future enhancement we could make the member list and count update live as invitations are accepted or people leave the group.

Reply:
Will look into it. Also from one of your prev suggestions, we still need to add some sort of signal/message when a user joins/leaves a group...so, will def work on this soon